src/modules/catalog.js

```diff
@@ -851,18 +853,18 @@ class Catalog {
    *
    * @function addItemGroups
    * @param {object} parameters - Additional parameters for item group details
-   * @param {object[]} parameters.item_groups - A list of item groups
+   * @param {object[]} parameters.itemGroups - A list of item groups
    * @param {object} [networkParameters] - Parameters relevant to the network request
    * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
    * @returns {Promise}
    * @see https://docs.constructor.io/rest_api/item_groups
    * @example
    * constructorio.catalog.addItemGroups({
-   *     item_groups: [
+   *     itemGroups: [
    *         {
    *             id: 'subcat_12891',
    *             name: 'Hoodies & Sweaters',
-   *             parent_id: 'cat_49203',
+   *             parentId: 'cat_49203',
```

Comment:
What do we think about leaving parentId as snake cased?

I was thinking that we would just camel case everything that we have explicitly documented as a parameter in our jsdoc comment, leave everything else as is.

Reply:
We explicitly document it under `addItemGroup` and I think it would be weird if one uses `parentId` and one uses `parent_id`. We might consider dropping `addItemGroup` function completely in favor of `addItemGroups` 🤔

Reply:
The nested values do get a little weird here...

It looks like we're leaving it as snake cased in [addRedirectRule](https://github.com/Constructor-io/constructorio-node/pull/139/files#r1123933057). 🤔 Maybe dropping `addItemGroup` like @esezen mentioned might be the way to go. We should review this with the team and get everyone's opinion.

Reply:
Sounds like we want to keep this as camel cased 👍